core/shared/src/main/scala/scalaz/zio/Clock.scala

```diff
@@ -4,7 +4,7 @@ package scalaz.zio
 import java.util.concurrent.TimeUnit
 import scala.concurrent.duration._
 
-trait Clock {
+trait Clock extends Serializable {
```

Comment:
I'd look at what the `Live` clock captures to make sure it is indeed serializable.

Reply:
Live captures ```nanotime``` which itself is serializable. Also confirmed by unit test.